data/outport/outportBlock.proto

```diff
@@ -15,41 +15,76 @@ import "github.com/multiversx/mx-chain-core-go/data/rewardTx/rewardTx.proto";
 import "github.com/multiversx/mx-chain-core-go/data/alteredAccount/alteredAccount.proto";
 
 message OutportBlock {
-  bytes                   HeaderBytes = 1 [(gogoproto.jsontag) = "headerBytes,omitempty"];
-  string                  HeaderType = 2 [(gogoproto.jsontag) = "headerType,omitempty"];
-  bytes                   HeaderHash = 3 [(gogoproto.jsontag) = "headerHash,omitempty"];
-  Body                    Body = 4 [(gogoproto.jsontag) = "body,omitempty"];
-  repeated AlteredAccount AlteredAccounts = 5 [(gogoproto.jsontag) = "alteredAccounts,omitempty"];
-  map<string, TxInfo>     Transactions = 6 [(gogoproto.jsontag) = "transactions,omitempty"];
-  map<string, SCRInfo>    SmartContractResults = 7 [(gogoproto.jsontag) = "smartContractResults,omitempty"];
-  map<string, RewardInfo> Rewards = 8 [(gogoproto.jsontag) = "rewards,omitempty"];
-  map<string, Receipt>    Receipts = 9 [(gogoproto.jsontag) = "receipts,omitempty"];
-  map<string, TxInfo>     InvalidTxs = 10 [(gogoproto.jsontag) = "invalidTxs,omitempty"];
-  map<string, Log>        Logs = 11 [(gogoproto.jsontag) = "logs,omitempty"];
-  repeated uint64         SignersIndexes = 12 [(gogoproto.jsontag) = "signersIndexes,omitempty"];
-  uint64                  HighestFinalBlockNonce = 13 [(gogoproto.jsontag) = "highestFinalBlockNonce"];
-  bytes                   HighestFinalBlockHash = 14 [(gogoproto.jsontag) = "highestFinalBlockHash,omitempty"];
+  HeaderBody                   HeaderBody             = 1 [(gogoproto.jsontag) = "headerBody,omitempty"];
+  TransactionPool              TransactionPool        = 2 [(gogoproto.jsontag) = "transactionPool,omitempty"];
+  HeaderGasConsumption         HeaderGasConsumption   = 3 [(gogoproto.jsontag) = "headerGasConsumption,omitempty"];
+  map <string, AlteredAccount> AlteredAccounts        = 4 [(gogoproto.jsontag) = "alteredAccounts,omitempty"];
+  repeated string              NotarizedHeadersHashes = 5 [(gogoproto.jsontag) = "notarizedHeadersHashes,omitempty"];
+  uint32                       NumberOfShards         = 6 [(gogoproto.jsontag) = "numberOfShards"];
+  bool                         IsImportDB             = 7 [(gogoproto.jsontag) = "isImportDB"];
+  repeated uint64              SignersIndexes         = 8 [(gogoproto.jsontag) = "signersIndexes,omitempty"];
+  uint64                       HighestFinalBlockNonce = 9 [(gogoproto.jsontag) = "highestFinalBlockNonce"];
+  bytes                        HighestFinalBlockHash  = 10 [(gogoproto.jsontag) = "highestFinalBlockHash,omitempty"];
+}
+
+message HeaderBody {
```

Comment:
the name `HeaderBody` is misleading in terms of other namings we are using since we already have `header` and `body` in the terminology. Might rename it to `BlockData`

Reply:
Refactored as suggested